### PR TITLE
Modifying AM and IAM accelerator modules

### PR DESCRIPTION
**Pull request by Ashi1993**
## Modifying AM and IAM accelerator modules

> Explain  in a few lines the purpose of this pull request

**Issue link:** *required*

**Doc Issue:** *Optional, link issue from [documentation repository](https://github.com/wso2/docs-ob/issues)*

**Applicable Labels:** *Spec, product, version, type (specify requested labels)*

------

### Development Checklist

1. [ ] Build complete solution with pull request in place.
2. [ ] Ran checkstyle plugin with pull request in place.
3. [ ] Ran Findbugs plugin with pull request in place.
4. [ ] Ran FindSecurityBugs plugin and verified report.
5. [ ] Formatted code according to WSO2 code style.
6. [ ] Have you verified the PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?
7. [ ] Migration scripts written (if applicable).
8. [ ] Have you followed secure coding standards in [WSO2 Secure Engineering Guidelines](http://wso2.com/technical-reports/wso2-secure-engineering-guidelines)?

### Testing Checklist

1. [ ] Written unit tests.
2. [ ] Verified tests in multiple database environments (if applicable).
3. [ ] Tested with BI enabled  (if applicable).

### CODE REVIEW

File: financial-services-accelerator/accelerators/fs-is/carbon-home/dbscripts/financial-services/consent/h2.sql [hunk 1]
```diff
@@ -64,39 +64,20 @@ CREATE TABLE IF NOT EXISTS OB_CONSENT_STATUS_AUDIT (
...
-CREATE TABLE IF NOT EXISTS OB_CONSENT_ATTRIBUTE (
+CREATE TABLE IF NOT EXISTS FS_CONSENT_ATTRIBUTE (
   CONSENT_ID        VARCHAR(255) NOT NULL,
   ATT_KEY           VARCHAR(255) NOT NULL,
   ATT_VALUE         VARCHAR(255) NOT NULL,
   PRIMARY KEY(CONSENT_ID, ATT_KEY),
-  CONSTRAINT FK_OB_CONSENT_ATTRIBUTE FOREIGN KEY (CONSENT_ID) REFERENCES OB_CONSENT (CONSENT_ID)
-);
-
-CREATE TABLE IF NOT EXISTS OB_THROTTLE_DATA (
```
Comment (Akila94) [inline financial-services-accelerator/accelerators/fs-is/carbon-home/dbscripts/financial-services/consent/h2.sql:85]: What is the reason to remove these tables?

Comment (Ashi1993) [reply financial-services-accelerator/accelerators/fs-is/carbon-home/dbscripts/financial-services/consent/h2.sql:85]: We are not planning to add these features to Accelerator 4

File: financial-services-accelerator/accelerators/fs-is/carbon-home/repository/components/dropins/classmate-1.5.1.jar
Comment (anjuchamantha) [inline financial-services-accelerator/accelerators/fs-is/carbon-home/repository/components/dropins/classmate-1.5.1.jar:1]: Why are we adding jars to the repo, can't we have them at compile time?

Comment (Ashi1993) [reply financial-services-accelerator/accelerators/fs-is/carbon-home/repository/components/dropins/classmate-1.5.1.jar:1]: We are adding these jars to the base products, thats why we kept this

**Review by Akila94 — APPROVED** [review]

**Review by anjuchamantha — APPROVED** [review]

**Pull request merged**